index.js

```diff
@@ -55,14 +55,17 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var isRelativeRoot = this.options.path && !path.isAbsolute(this.options.path)
```

Comment:
I'm relatively new to Javasript, so forgive me if this comment is misplaced, but I found that to get this to work I had to use `self` here and throughout these code changes instead of `this`. We use self below when checking for the `publicPath`, so I believe it's correct in the below context, but it also seems to be required here.

My changes are here https://github.com/veryhappythings/webpack-bundle-tracker/commit/8bb4d378f66ad6072a9db35fe3218aa13043533f and work for me - using Node v9.11.1